tests/Unit/Domain/Regulation/MeasureTest.php

```diff
@@ -35,8 +36,8 @@ public function testMeasure(): void
         $this->assertSame($date, $measure->getCreatedAt());
         $this->assertEmpty($measure->getPeriods()); // Automatically set by Doctrine
 
-        $measure->update(MeasureTypeEnum::ALTERNATE_ROAD->value);
-        $this->assertSame(MeasureTypeEnum::ALTERNATE_ROAD->value, $measure->getType());
+        $measure->update(MeasureTypeEnum::SPEED_LIMITATION->value, 50);
+        $this->assertSame(MeasureTypeEnum::SPEED_LIMITATION->value, $measure->getType());
```

Comment:
On peut aussi vérifier la nouvelle maxSpeed

```suggestion
        $this->assertSame(MeasureTypeEnum::SPEED_LIMITATION->value, $measure->getType());
        $this->assertSame(50, $measure->getMaxSpeed());
```